app/code/Magento/PageBuilder/view/frontend/web/css/source/_icons.less

```diff
@@ -16,6 +16,7 @@
         src: url('@{icons-pagebuilder__font-path}.eot?#iefix') format('embedded-opentype'),
         url('@{icons-pagebuilder__font-path}.woff') format('woff'),
         url('@{icons-pagebuilder__font-path}.ttf') format('truetype');
+        font-display: swap;
```

Comment:
Nice one!

Personally, I'd use `font-display: block` in the same way `magento/theme-frontend-blank/web/css/source/_icons.less` implements. Unlike text-based fonts where you're showing _something_ before progressively "swapping" to the designated font, icons don't have anything useful to show.

There's a [CSS Tricks](https://css-tricks.com/hey-hey-font-display) article that still stands up today that does a far better job of explaining than I could hope to (also covers the icon font use case).

In addition, I would recommend adding this to the _top_ with the other `font-*` properties for improved readability.

I don't know why this implementation doesn't use the `.lib-font-face()` wrapper, `font-style` and `font-weight` don't use `@font-style__base` and `@font-weight__regular` accordingly, etc.